webapp/app/modules/rooms/rooms.selectors.js

```diff
@@ -2,12 +2,25 @@ import { createSelector } from 'reselect';
 import { fromJS, Map } from 'immutable';
 import { complement, isNil } from 'ramda';
 
+import { selectUserUid } from '../userAuth/userAuth.selectors';
+
 
 export const selectRooms = (state) => state.get('rooms', Map());
 
 export const selectRoomsList = createSelector(
-  selectRooms,
-  (state) => state.get('roomsList', Map()),
+  selectRooms, selectUserUid,
+  (rooms, userUid) => rooms.get('roomsList', Map())
+    .map((room) => room.set('isActive', room.getIn(['value', 'users', userUid], false))),
```

Comment:
room.update('isActive', !!path(['value', 'users', userUid])))

Reply:
no tak nie do końca, bo:

1. funkcja przekazywana do update przyjmuje jako parametr wartość propa, a nie obiekt
2. to jest immutable więc path tego nie ogarnie